snuba/migrations/operations.py

```diff
@@ -75,6 +75,7 @@ def __init__(
 
     def format_sql(self) -> str:
         columns = ", ".join([col.for_schema() for col in self.__columns])
-        engine = self.__engine.get_sql()
+        single_node = get_cluster(self._storage_set).is_single_node()
```

Comment:
You can inline this call into line 79. The variable is not used anywhere else and it would not compromise readability, so there is one less detail to keep in mind.